CMakeLists.txt

```diff
@@ -114,6 +114,8 @@ set(SYSREPO_UMASK "00000" CACHE STRING "Umask used for any files created by sysr
 # global group settings
 set(SYSREPO_GROUP "" CACHE STRING "System group that will own all sysrepo-related files. If empty, the specific process group will be kept.")
 
+set(NACM_RECOVERY_UID 0 CACHE STRING "NACM recovery session UID that has unrestricted access")
```

Comment:
Not sure if it needs to be separate, I would just unite it with `SYSREPO_SUPERUSER_UID`.

Reply:
done